### ISSUE TITLE
"github" sub-dependency is not installed

### ISSUE THREAD

**Issue by kanongil** [npm/npm#8062]
When I install `ember-cli` successfully, the `github` dependency is never installed.

```sh
$ npm install ember-cli
$ npm ls
```

Result:
```
…
npm ERR! missing: github@^0.2.3, required by bower@1.4.1
```

Tested on `npm@2.9.0` & `npm@2.8.4`.

**Comment by akappen** [comment]
I think you may be hitting this issue:

https://github.com/ember-cli/ember-cli/issues/3885

**Comment by danbell** [comment]
+1 Just hit this as well. Adding `github` as dependency is a workaround until this is fixed, or `ember-cli 0.2.4` is released.

**Comment by kanongil** [comment]
Same issue as #7887. Further discussion should probable be handled there.
